resources/frontend/wp-convertkit.js

```diff
@@ -1,34 +1,38 @@
 jQuery(document).ready(function($) {
 
     // Manage visit cookie
-    var subscriber_id = $.cookie( 'ck_subscriber_id' );
+    var subscriber_id = parseInt($.cookie('ck_subscriber_id')) ? parseInt($.cookie('ck_subscriber_id')) : false;
 
     if ( ! subscriber_id ) {
         subscriber_id = ckGetQueryVariable('ck_subscriber_id');
     }
 
-    /* Check if subscriber_id is valid and maybe do add tags */
-    $.ajax({
-        type: "POST",
-        data: {
-            action: 'ck_add_user_visit',
-            subscriber_id: subscriber_id,
-            url: document.URL
-        },
-        url: ck_data.ajaxurl,
-        success: function (response) {
-            var values = JSON.parse(response);
-            if ( 0 != values.subscriber_id) {
-                $.cookie('ck_subscriber_id', values.subscriber_id, {expires: 365, path: '/'});
-                ckRemoveSubscriberId( window.location.href );
+    // Only POST to admin-ajax.php if we have a subscriber_id to use...
+    // ...and the current post has a tag assigned to it
+    if ( subscriber_id && ck_data.post_has_tag ) {
+        /* Check if subscriber_id is valid and maybe do add tags */
+        $.ajax({
+            type: "POST",
+            data: {
+                action: 'ck_add_user_visit',
+                subscriber_id: subscriber_id,
+                url: document.URL
+            },
+            url: ck_data.ajaxurl,
+            success: function (response) {
+                var values = JSON.parse(response);
+                if ( 0 != values.subscriber_id) {
+                    $.cookie('ck_subscriber_id', values.subscriber_id, {expires: 365, path: '/'});
+                    ckRemoveSubscriberId( window.location.href );
+                }
             }
-        }
 
-    }).fail(function (response) {
-        if ( window.console && window.console.log ) {
-            console.log( "AJAX ERROR" + response );
-        }
-    });
+        }).fail(function (response) {
+            if ( window.console && window.console.log ) {
+                console.log( "AJAX ERROR" + response );
+            }
```

Comment:
Should that only be in development?

Reply:
Good question. It's been there this whole time, so for now I've left it in. I _think_ it could be somewhat useful in debugging a customer site if the tagging function is failing for them, since we'd be able to see that output ourselves when testing, or instruct them on how to look (ok, not likely).

My hunch is it was put there originally as a "well, if there's a failure we should do _something_".